cupy/math/window.py

```diff
@@ -1,9 +1,14 @@
 import numpy
 
+from cupy import array
+from cupy import empty
+from cupy import float64
+from cupy import util
```

Comment:
Do not directly import symbols from the namespace that are not modules
https://docs-cupy.chainer.org/en/stable/contribution.html#coding-guidelines

Use them as `cupy.array`